src/utility/time/TimeService.h

```diff
@@ -55,8 +59,13 @@ class TimeService
 #endif
 
   unsigned long getRemoteTime();
+
   static bool isTimeValid(unsigned long const time);
 
+private:
+  unsigned long _timezone_offset;
```

Comment:
the offset can be negative

Reply:
Nice catch! thanks

Reply:
fixed with cd9a6642f0904630e9d5d747e28439e06b313b20